templates/python/custom-copilot-rag-azure-ai-search/src/bot.py.tpl

```diff
@@ -0,0 +1,95 @@
+import asyncio
+from dataclasses import dataclass
+import json
+import os
+import sys
+import traceback
+from typing import Generic, TypeVar
+
+from botbuilder.core import MemoryStorage, TurnContext
+from teams import Application, ApplicationOptions, TeamsAdapter
+from teams.ai import AIOptions
+from teams.ai.models import AzureOpenAIModelOptions, OpenAIModel, OpenAIModelOptions
+from teams.ai.planners import ActionPlanner, ActionPlannerOptions
+from teams.ai.prompts import PromptManager, PromptManagerOptions
+from teams.ai.actions import ActionTypes
+from teams.state import TurnState
+
+from .AzureAISearchDataSource import AzureAISearchDataSource, AzureAISearchDataSourceOptions
+from .config import Config
+
+config = Config()
+
+class ConversationState:
+    pass
+
+T = TypeVar('T')
+
+class TurnState(Generic[T]):
+    pass
+
+ApplicationTurnState = TurnState[ConversationState]
```

Comment:
Is it possible to just use the `TurnState` from Teams-AI (`from teams.state import TurnState`)? Seems the code does not customize the state.

Reply:
will check